<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GobangBoard</class>
 <widget class="QWidget" name="GobangBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="connServerBtn">
      <property name="text">
       <string>进入大厅</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器IP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serverIPLE">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serverPortLE">
      <property name="text">
       <string>8888</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>640</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>2</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天大厅</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>231</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>610</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>670</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="sendMsg">
      <property name="text">
       <string>发送给对手</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="broadcastBtn">
      <property name="text">
       <string>广播到大厅</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="loseBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>673</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>认输</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="totalLCD">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>670</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="gameStatusLB">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>欢迎来玩五子棋</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>650</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>步时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>680</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>当总时耗尽时，您会自动断开和服务器的链接</string>
   </property>
   <property name="text">
    <string>总时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rivalLB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>680</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aharoni</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>对局还未开始</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MYLB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>650</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>您还没有开始游戏！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ranklistBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>673</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查看排行榜</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
